### PR TITLE
css-layout-classes complete

### PR DESCRIPTION
**Pull request by luiderek**
image

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  <link rel="stylesheet" href="styles.css">
+  <title>css layout classes</title>
+</head>
+
+<body>
+
+  <div class="container">
+
+    <div class="row">
+      <h1>Choose your character!</h1>
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:17]: only addition would be a full width column here around the h1

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Great work, small update needed

File: css-layout-classes/index.html (hunk 1 above)
Comment (luiderek) [reply css-layout-classes/index.html]: gotcha!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**